meta-resin-common/recipes-support/resin-expand/resin-filesystem-expand/resin-filesystem-expand

```diff
@@ -5,7 +5,7 @@ set -e
 DATA_PARTITION=/dev/disk/by-label/resin-data
 
 echo -n "Expand ext4 partition on $DATA_PARTITION... "
-e2fsck -y -f $DATA_PARTITION
+e2fsck -y $DATA_PARTITION
```

Comment:
@floion every boot is affected by this flag. The -f always forces a full fsck even if the dirty bit is not marked.